docs/configuration-reference/components/headlamp.md

```diff
@@ -0,0 +1,70 @@
+# Headlamp configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+[Headlamp](https://github.com/kinvolk/headlamp) is an easy-to-use and versatile
+dashboard for Kubernetes.
+
+It has a clean and modern UI, it is vendor independent, generic, and supports
+the most common operations for Kubernetes clusters.
+
+## Prerequisites
+
+* A Kubernetes cluster accessible via `kubectl`.
+
+* An ingress controller such as [Contour](contour.md) for HTTP ingress.
+
+* [cert-manager](cert-manager.md) to generate TLS certificates.
+
+## Configuration
+
+```tf
+# headlamp.lokocfg
+
+component "headlamp" {
+  ingress {
+    host                       = "headlamp.example.lokomotive-k8s.org"
+    class                      = "contour"
+    certmanager_cluster_issuer = "letsencrypt-production"
+  }
+}
+```
+
+## Attribute reference
+
+Table of all the arguments accepted by the component.
+
+Example:
+
+| Argument                             | Description                                                                                                                                   | Default                  | Type   | Required |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|--:-:---------------------|--:-:---|--:-:-----|
+| `namespace`                          | Namespace where Headlampwill be installed.                                                                                                    | "kube-system"            | string | false    |
+| `ingress`                            | Configuration block for exposing Headlamp through an ingress resource.                                                                        | -                        | block  | false    |
```

Comment:
I think `Ingress` should be capitalized if we talk about the resource. "ingress" as "incoming traffic" would be fine not-capitalized I guess.
```suggestion
| `ingress`                            | Configuration block for exposing Headlamp through an Ingress resource.                                                                        | -                        | block  | false    |
```